src/jobs/deploy_to_selfhosted.yml

```diff
@@ -0,0 +1,39 @@
+description: "A job to deploy latest dev components to self-hosted"
+
+executor: do
+
+steps:
+  - deploy:
+      name: Trigger workflow to deploy dev components
+      command: |
+        function get_workflow_status {
+            response=$(curl -s -H 'Accept: application/json' -H 'Content-type: application/json' "https://circleci.com/api/v2/pipeline/$workflowID/workflow" -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN" | jq '.items[0].status' | sed "s/\"//g")
+            echo $response
+        }
+
+        response=$(curl -XPOST -s -H 'Accept: application/json' -H 'x-attribution-login: string' -H 'x-attribution-actor-id: string' -H 'Content-type: application/json' -d '{"branch": "master"}' 'https://circleci.com/api/v2/project/gh/codacy/chart/pipeline' -H "Circle-Token: $CIRCLE_CI_GITHUB_TOKEN")
+        workflowState=$(echo $response | jq '.state' | sed 's/\"//g')
```

Comment:
This sed after the jq is to remove the quotes from jq?
If yes you can do that natively in jq with `--raw` or something similar

Reply:
I didn't know that was a thing!!! that's a life saver! \o/ 
Thanks!